cmd/tag.go

```diff
@@ -0,0 +1,164 @@
+	addCmd := &cobra.Command{
+		Use:   "add [flags] SBOM_ID TAGS...",
+		Short: "Add tags to a document",
+		Long:  "Add tags to a document",
+		Args:  cobra.MinimumNArgs(tagAddMinArgs),
+		Run: func(cmd *cobra.Command, args []string) {
+			backend := backendFromContext(cmd)
+
+			defer backend.CloseClient()
+
+			document, err := backend.GetDocumentByIDOrAlias(args[0])
+			if err != nil {
+				backend.Logger.Fatalf("failed to get document: %v", err)
+			}
+
+			if document == nil {
+				backend.Logger.Fatal(errDocumentNotFound)
+			}
+
+			if err := backend.AddAnnotations(document.Metadata.Id, db.BomctlAnnotationTag, args[1:]...); err != nil {
+				backend.Logger.Fatalf("failed to add tags: %v", err)
+			}
+		},
+	}
+
+	return addCmd
+}
+
+func tagClearCmd() *cobra.Command {
+	clearCmd := &cobra.Command{
+		Use:   "clear [flags] SBOM_ID...",
+		Short: "Clear the tags of a document",
+		Long:  "Clear the tags of a document",
+		Args:  cobra.ExactArgs(tagClearExactArgs),
+		Run: func(cmd *cobra.Command, args []string) {
+			backend := backendFromContext(cmd)
+
+			defer backend.CloseClient()
+
+			document, err := backend.GetDocumentByIDOrAlias(args[0])
+			if err != nil {
+				backend.Logger.Fatalf("failed to get document: %v", err)
+			}
+
+			if document == nil {
+				backend.Logger.Fatal(errDocumentNotFound)
+			}
+
+			annotationsToRemove, err := backend.GetDocumentAnnotations(document.Metadata.Id, db.BomctlAnnotationTag)
+			if err != nil {
+				backend.Logger.Fatalf("failed to clear tags: %v", err)
+			}
+
+			tagsToRemove := make([]string, len(annotationsToRemove))
+			for idx := range annotationsToRemove {
+				tagsToRemove[idx] = annotationsToRemove[idx].Value
+			}
+
+			err = backend.RemoveAnnotations(document.Metadata.Id, db.BomctlAnnotationTag, tagsToRemove...)
+			if err != nil {
+				backend.Logger.Fatalf("failed to clear tags: %v", err)
+			}
+		},
+	}
+
+	return clearCmd
+}
+
+func tagListCmd() *cobra.Command {
+	listCmd := &cobra.Command{
+		Use:     "list [flags] SBOM_ID",
+		Aliases: []string{"ls"},
+		Short:   "List the tags of a document",
+		Long:    "List the tags of a document",
+		Args:    cobra.ExactArgs(tagListExactArgs),
+		Run: func(cmd *cobra.Command, args []string) {
+			backend := backendFromContext(cmd)
+
+			defer backend.CloseClient()
+
+			document, err := backend.GetDocumentByIDOrAlias(args[0])
+			if err != nil {
+				backend.Logger.Fatal("Failed to get document", "err", err)
+			}
+
+			if document == nil {
+				backend.Logger.Fatal(errDocumentNotFound)
+			}
+
+			annotations, err := backend.GetDocumentAnnotations(document.Metadata.Id, db.BomctlAnnotationTag)
+			if err != nil {
+				backend.Logger.Fatal("Failed to get document tags", "err", err)
+			}
+
+			for _, annotation := range annotations {
+				fmt.Fprintln(os.Stdout, annotation.Value)
+			}
+		},
+	}
+
+	return listCmd
+}
+
+func tagRemoveCmd() *cobra.Command {
+	removeCmd := &cobra.Command{
+		Use:     "remove [flags] SBOM_ID",
```

Comment:
```suggestion
		Use:     "remove [flags] SBOM_ID tag...",
```
This one needs a existing tag specified if I'm correct, not sure if it can handle more than one at a time

Reply:
It can. Nice catch